src/input_file.rst

```diff
@@ -1112,6 +1169,24 @@ Note that some tags, or entries for tags, are only used in certain ensembles (e.
     Fugacity  Si     0.0
     Fugacity  O      0.0
 
+  - Value 1: String - The residue to apply this fugacity.
+  - Value 2: List of Doubles - A list of the fugacity values in bar.
+
+  .. note:: 
+    - For binary systems, include multiple copies of the tag (one per residue kind).
+    - If there is a molecule kind that cannot be transfer between boxes (in PDB file the beta value is set to 1.00 or 2.00) an arbitrary value e.g. 0.00 can be assigned to the residue name.
+
+  .. code-block:: text
+
+    #################################
+    # Mol.  Name Fugacity (bar)
+    #################################
+    Fugacity  ISB   10.0     12.00     14.00     16.00
+    Fugacity  Si     0.0       1.0       2.0       3.0
+    Fugacity  O      0.0       0.5      0.75       1.0
```

Comment:
Keep `Si` and `O` Fugacity value zero.

Reply:
I changed Si to non-zero to illustrate the flexibility of multisim, but left O to 0.0 to illustrate consistently how to for a molecule to not be transferred.